packages/babel-preset-env/src/polyfills/babel-7-plugins.cjs

```diff
@@ -0,0 +1,9 @@
+// TODO(Babel 8): Remove this file
```

Comment:
I rewrote the files in this folder to CJS so that they can easily be conditionally required.